src/interface/src/app/map/project-card/project-card.component.ts

```diff
@@ -1,13 +1,77 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { Feature, Geometry } from 'geojson';
 
+interface Boundary {
+  shape_name: string;
+}
+
+interface Project {
+  PROJECT_NAME?: string;
```

Comment:
is there a reason why these are all capitalized?

Reply:
Not sure, it's how they're coming from the backend (and I assume straight from CalMapper). JSON is case-sensitive so we'd have to convert each of these fields if we wanted to use typical camel case.